region/maridia/outer/Boyon Gate Hall.json

```
@@ -257,13 +265,21 @@
         "h_canArtificialMorphIBJ",
         "canOffScreenMovement"
       ],
+      "clearsObstacles": ["A"],
       "flashSuitChecked": true,
-      "note": "The gate will not spawn in indirect g-mode and is freely passable."
+      "note": [
+        "The gate will not spawn in indirect g-mode and is freely passable.",
```

```suggestion
        "The gate will not spawn in indirect G-mode and is freely passable.",
```
I know this is inconsistent in other places too, and it's not important or in scope of this PR. Just a suggestion for starting to standardize on the capitalization used